<launch>
  <include file="$(find tiago_iaslab_simulation)/launch/start_simulation.launch">
    <arg name="world_name" value="iaslab_assignment1"/>
  </include>

  <include file="$(find tiago_iaslab_simulation)/launch/navigation.launch"></include>
  <include file="$(find tiago_iaslab_simulation)/launch/apriltag.launch"></include>

  <node name="apriltag_ids_generator_node" pkg="tiago_iaslab_simulation" type="apriltag_ids_generator_node"></node>
</launch>